review-notifications/src/Github.js

```diff
@@ -224,6 +257,28 @@ class Github extends Component {
     return listWithoutDuplicates;
   }
 
+  handleMoreLess(e) {
+    var div = e.target.previousSibling;
+    if (div.className === 'showLess') {
+      div.className = '';
+      e.target.innerHTML = 'show less';
+    } else {
+      div.className += 'showLess';
+      e.target.innerHTML = 'show more';
+    }
+  }
+
+  displayButtonOrNo(e) {
```

Comment:
imo this is a weird name, `handleDisplayButton` or `toggleDisplayButton` sounds better to me, but that's only an opinion